packages/sdks/web-component/src/lib/helpers/templates.ts

```diff
@@ -201,10 +201,26 @@ const setElementConfig = (
   if (!componentsConfig) {
     return;
   }
+  const { componentsDynamicAttrs, ...rest } = componentsConfig;
+
+  const configMap = Object.keys(rest).reduce((acc, componentName) => {
+    acc[`[name=${componentName}]`] = rest[componentName];
+    return acc;
+  }, {});
+
+  if (componentsDynamicAttrs) {
+    Object.keys(componentsDynamicAttrs).forEach((componentSelector) => {
+      const { attributes } = componentsDynamicAttrs[componentSelector];
```

Comment:
might want to avoid deconstruct here if componentsDynamicAttrs[componentSelector] is undefined this will fail